packages/peregrine/lib/talons/WishlistPage/useWishlistItem.js

```diff
@@ -23,19 +33,26 @@ const dialogs = {
  */
 export const useWishlistItem = props => {
     const { item, onOpenAddToCartDialog, wishlistId } = props;
+
     const {
         configurable_options: selectedConfigurableOptions = [],
         id: itemId,
         product
     } = item;
+
     const {
         configurable_options: configurableOptions = [],
+        __typename: productType,
         image,
         sku,
         stock_status: stockStatus
     } = product;
     const { label: imageLabel, url: imageURL } = image;
 
+    const isSupportedProductType = mergeSupportedProductTypes(
```

Comment:
Can you memoize this using a `useMemo`?

Reply:
`mergeOperations` is a the same of cost and is also nowhere with memo would recommend don't memo this as well

Reply:
If the inputs are not changing, why would it have merit to run the same logic on every render? Wouldn't it make sense to memorize it even if it is a simple calculation? 

Memoizing such calculation is a standard norm we follow in the code base.